Kernel/FileSystem/VirtualFileSystem.cpp

```diff
@@ -635,10 +635,12 @@ Custody& VFS::root_custody()
     return *m_root_custody;
 }
 
-KResultOr<NonnullRefPtr<Custody>> VFS::resolve_path(StringView path, Custody& base, RefPtr<Custody>* parent_custody, int options)
+KResultOr<NonnullRefPtr<Custody>> VFS::resolve_path(StringView path, Custody& base, RefPtr<Custody>* parent_custody, int options, int accumulator)
 {
     // FIXME: resolve_path currently doesn't deal with .. and . . If path is ../. and base is /home/anon, it returns
     //        /home/anon/../. instead of /home .
+    if (accumulator >= symlink_recursion_limit)
```

Comment:
`accumulator` is a rather opaque name. How about `symlink_recursion_level`?

Reply:
Ah good point, fixed :)